### PR TITLE
Update vertical pills example

### LINKED ISSUE #25706: Update vertical tabs nav example
The example code for navs using vertical tabs (https://getbootstrap.com/docs/4.0/components/navs/#javascript-behavior) leaves out that it internally creates a row and puts the links in one column and the tab content in another.  You have to inspect the code of the bootstrap documentation to see how it it actually works on that page.

It should really look similar to:

```html
<div class="bd-example bd-example-tabs">
  <div class="row">
    <div class="col-3">
      <div class="nav flex-column nav-pills" id="v-pills-tab" role="tablist" aria-orientation="vertical">
        <a class="nav-link active" id="v-pills-home-tab" data-toggle="pill" href="#v-pills-home" role="tab" aria-controls="v-pills-home" aria-selected="true">Home</a>
        <a class="nav-link" id="v-pills-profile-tab" data-toggle="pill" href="#v-pills-profile" role="tab" aria-controls="v-pills-profile" aria-selected="false">Profile</a>
        <a class="nav-link" id="v-pills-messages-tab" data-toggle="pill" href="#v-pills-messages" role="tab" aria-controls="v-pills-messages" aria-selected="false">Messages</a>
        <a class="nav-link" id="v-pills-settings-tab" data-toggle="pill" href="#v-pills-settings" role="tab" aria-controls="v-pills-settings" aria-selected="false">Settings</a>
      </div>
    </div>
    <div class="col-9">
      <div class="tab-content" id="v-pills-tabContent">
        <div class="tab-pane fade show active" id="v-pills-home" role="tabpanel" aria-labelledby="v-pills-home-tab">...</div>
        <div class="tab-pane fade" id="v-pills-profile" role="tabpanel" aria-labelledby="v-pills-profile-tab">...</div>
        <div class="tab-pane fade" id="v-pills-messages" role="tabpanel" aria-labelledby="v-pills-messages-tab">...</div>
        <div class="tab-pane fade" id="v-pills-settings" role="tabpanel" aria-labelledby="v-pills-settings-tab">...</div>
      </div>
    </div>
  </div>
</div>
```

### PR DESCRIPTION
**Pull request by Varunram**
Fixes #25706

### CODE REVIEW

File: docs/4.0/components/navs.md [hunk 1]
```diff
@@ -479,17 +479,25 @@ And with vertical pills.
...
-  <a class="nav-link" id="v-pills-profile-tab" data-toggle="pill" href="#v-pills-profile" role="tab" aria-controls="v-pills-profile" aria-selected="false">Profile</a>
-  <a class="nav-link" id="v-pills-messages-tab" data-toggle="pill" href="#v-pills-messages" role="tab" aria-controls="v-pills-messages" aria-selected="false">Messages</a>
-  <a class="nav-link" id="v-pills-settings-tab" data-toggle="pill" href="#v-pills-settings" role="tab" aria-controls="v-pills-settings" aria-selected="false">Settings</a>
-</div>
-<div class="tab-content" id="v-pills-tabContent">
-  <div class="tab-pane fade show active" id="v-pills-home" role="tabpanel" aria-labelledby="v-pills-home-tab">...</div>
-  <div class="tab-pane fade" id="v-pills-profile" role="tabpanel" aria-labelledby="v-pills-profile-tab">...</div>
-  <div class="tab-pane fade" id="v-pills-messages" role="tabpanel" aria-labelledby="v-pills-messages-tab">...</div>
-  <div class="tab-pane fade" id="v-pills-settings" role="tabpanel" aria-labelledby="v-pills-settings-tab">...</div>
+<div class="bd-example bd-example-tabs">
```
Comment (patrickhlauke) [inline docs/4.0/components/navs.md]: remove the outer `<div class="bd-example...">`  ... this is only specific to our documentation,

File: docs/4.0/components/navs.md [hunk 2]
```diff
@@ -479,17 +479,25 @@ And with vertical pills.
...
+    </div>
+    <div class="col-9">
+      <div class="tab-content" id="v-pills-tabContent">
+        <div class="tab-pane fade show active" id="v-pills-home" role="tabpanel" aria-labelledby="v-pills-home-tab">...</div>
+        <div class="tab-pane fade" id="v-pills-profile" role="tabpanel" aria-labelledby="v-pills-profile-tab">...</div>
+        <div class="tab-pane fade" id="v-pills-messages" role="tabpanel" aria-labelledby="v-pills-messages-tab">...</div>
+        <div class="tab-pane fade" id="v-pills-settings" role="tabpanel" aria-labelledby="v-pills-settings-tab">...</div>
+      </div>
+    </div>
+  </div>
```
Comment (patrickhlauke) [inline docs/4.0/components/navs.md]: remove the matching closing `</div>` as per the previous comment

**Review by patrickhlauke — CHANGES_REQUESTED** [review]
apart from the extraneous `<div class="bd-example ...">` this looks good to me. it doesn't change anything fundamental in the structure (ed: in terms of accessibility i mean), but clarifies the two-column approach used to generate the actual vertical tabs seen in the docs

File: docs/4.0/components/navs.md (hunk 1 above)
Comment (Varunram) [reply docs/4.0/components/navs.md]: Oh, sorry, didn't know that. Will remove.

**Comment by Varunram** [comment]
Requested changes made. Please let me know if there's something else to add.

**Comment by patrickhlauke** [comment]
@Varunram you accidentally removed it from the live code rather than the sample code...went in to fix that up for ya ;)

**Review by patrickhlauke — APPROVED** [review]

**Comment by Varunram** [comment]
Oh my, should have slept properly instead of rushing to fix that up, thanks! :)

**Pull request merged**